docs/wip/exchange-onboarding.md

```diff
@@ -0,0 +1,352 @@
+
+    curl -X GET https://localhost:8090/api/v1/wallets/{{walletId}} \
+         -H "Accept: application/json; charset=utf-8"              \
+         --cacert ./scripts/tls-files/ca.crt                       \
+
+## Receiving Money
+
+To receive money from other users you should provide your address. This address can be obtained from an account. Each wallet contains at least one account, you can think of account as a pocket inside of your wallet. Besides, you can view all existing accounts of a wallet by using the GET /api/v1/wallets/{{walletId}}/accounts endpoint as follow:
+
+    curl -X GET https://localhost:8090/api/v1/wallets/{{walletId}}/accounts?page=1&per_page=10 \
+         -H "Accept: application/json; charset=utf-8"                                          \
+         --cacert ./scripts/tls-files/ca.crt                                                   \
+Since you have, for now, only a single wallet, you’ll see something like this:
+
+    {
+        "status": "success",
+        "data": [
+            {
+                "index": 2147483648,
+                "addresses": [
+                    "DdzFFzCqrh...fXSru1pdFE"
+                ],
+                "amount": 0,
+                "name": "Initial account",
+                "walletId": "Ae2tdPwUPE...8V3AVTnqGZ"
+            }
+        ],
+        "meta": {
+            "pagination": {
+                "totalPages": 1,
+                "page": 1,
+                "perPage": 10,
+                "totalEntries": 1
+            }
+        }
+    }
+
+Each account has at least one address, all listed under the addresses field. You can communicate one of these addresses to receive money on the associated account.
+
+## Sending Money
+
+In order to send money from one of your account to another address, you can create a new payment transaction using the POST /api/v1/transactions endpoint as follow:
+
+    curl -X POST https://localhost:8090/api/v1/transactions \
+         -H "Content-Type: application/json; charset=utf-8" \
+         -H "Accept: application/json; charset=utf-8"       \
+         --cacert ./scripts/tls-files/ca.crt                \
+         -d '{                                              \
+      "destinations": [{                                    \
+        "amount": 14,                                       \
+        "address": "A7k5bz1QR2...Tx561NNmfF"                \
+      }],                                                   \
+      "source": {                                           \
+        "accountIndex": 0,                                  \
+        "walletId": "Ae2tdPwUPE...8V3AVTnqGZ"               \
+      }                                                     \
+    }'
+
+Note that, in order to perform a transaction, you need to have some existing coins on the source account! Beside, the Cardano API is designed to accomodate multiple recipients payments out-of-the-box; notice how destinations is a list of addresses.
```

Comment:
s/some existing/enough/

s/Beside,//

> list of addresses

"and amounts"

Reply:
completely agree, but should probably be fixed in API docs, this was a straight copy/paste from there.

Reply:
Can you follow up on making sure the API docs are updated as well? If the source is in this repo, you can just include it in this PR.

Reply:
@disassembler Indeed we generate that "Getting started" from a Haskell interpolated text, available here: https://github.com/input-output-hk/cardano-sl/blob/develop/wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs#L221